crates/parry3d/examples/project_point3d.rs

```diff
@@ -0,0 +1,91 @@
+use macroquad::prelude::*;
+use nalgebra::Vector3;
+use parry3d::math::Isometry;
+use parry3d::query::PointQuery;
+use parry3d::shape::{Cuboid, TriMesh, TriMeshFlags};
+
+mod common_macroquad;
+use common_macroquad::*;
+
+#[macroquad::main("parry3d::query::PlaneIntersection")]
+async fn main() {
+    let trimesh = Cuboid::new(Vector3::new(0.2, 0.5, 1.0)).to_trimesh();
+
+    let mesh = mquad_mesh_from_points(
+        &trimesh,
+        Vec3::new(1f32, 3f32, 3f32),
+        Color::from_rgba(200, 200, 200, 150),
+    );
+    let (points, indices) = trimesh;
+    let trimesh = TriMesh::with_flags(points, indices, TriMeshFlags::ORIENTED);
+    for _i in 1.. {
+        clear_background(BLACK);
+
+        let elapsed_time = get_time() as f32;
+        let slow_elapsed_time = elapsed_time / 3.0;
+
+        let point_to_project = lissajous_3d(slow_elapsed_time);
+        let projected_point = trimesh.project_point(
+            &Isometry::identity(),
+            &na_from_mquad(point_to_project),
+            true,
+        );
```

Comment:
Since the transform is the identity, we can call `project_local_point` here:
```suggestion
        let projected_point = trimesh.project_local_point(
            &na_from_mquad(point_to_project),
            true,
        );
```